lib/web_ui/test/engine/history_test.dart

```diff
@@ -276,3 +312,37 @@ Future<void> systemNavigatorPop() {
   );
   return completer.future;
 }
+
```

Comment:
nit: may be add a doc on top of the class.

Reply:
Done